qualang_tools/external_frameworks/qcodes/README.md

```diff
@@ -0,0 +1,165 @@
+# QCoDeS drivers
+
+This subpackage contains drivers to set the OPX as a QCoDeS instrument and synchronize external parameter sweeps with 
+sequences running on the OPX using the QCoDeS do"nd" methods. 
+Since the OPX contains an FPGA, it can perform complex sequences, acquire data with different formats and 
+sweep parameters on his own, which complexifies a bit the drivers compared to other instruments.
```

Comment:
```suggestion
Since the OPX runs on a unique pulse processor, it can perform complex sequences that QCoDeS do not natively support. For example, acquire data with different formats and sweep parameters on his own.
To more fully support the OPX capabilities, this driver is more complex compared to other QCoDeS instruments' drivers.
```
Something along these lines.
Note that also QBlox and ZI have an FPGA, it's what you do with it :)

Reply:
Great I did the modification :)